security.md

```diff
@@ -149,6 +140,16 @@ Using git would allow you to add outsource/freelance developers for a limited ti
 
 * There are extra protection products on top of an antivirus called EDR (Cyberreason, BlackCobalt) but these are usually costly.  
 
+### Hash your customer's passwords with a proper password hashing function
+
+
+If you can afford it, use a [third party authentication service](#customer-users-management) to handle password storage, password management and password recovery. 

```

Comment:
You loose points with me for using the [non-word "Architected"](https://twitter.com/mattkocaj/status/786418873469763584) 😛 http://developer.okta.com/product/